checkbox-core-snap/series16/snap/snapcraft.yaml

```diff
@@ -135,14 +145,25 @@ parts:
       - PYTHONPATH: $SNAPCRAFT_PART_INSTALL/usr/lib/python3/dist-packages:$PYTHONPATH
     override-stage: |
       snapcraftctl stage
-      sed -i 's|except OSError:  # Command not found|except subprocess.CalledProcessError:  # Command not found|g' lib/python3.*/site-packages/distro.py
+      sed -i 's|except OSError:  # Command not found|except subprocess.CalledProcessError:  # Command not found|g' lib/python3.*/site-packages/**/**/distro.py
```

Comment:
this absolutely warrants a big paragraph of explanation :D

Reply:
I have no idea why it is there, I have just updated it because the path of that lib has changed